### ISSUE TITLE
Export Does NOT Create File

### ISSUE THREAD

**Issue by GuardianMajor** [AdAway/AdAway#776]
I have been trying to EXPORT my lists but each time it says that a file `adway-export.txt` has been created on the SD Card, but no such file is ever generated.

Anyone know why this is not working?

**Comment by PerfectSlayer** [comment]
Hi @GuardianMajor 

Check the #670 to understand your issue.
You have to manually give the access local storage permission.

Regards.

**Comment by GuardianMajor** [comment]
@PerfectSlayer Thank you for pointing that out. I appreciate it. I was aware of that issue already.

What puzzles me is that I already took that into consideration and DID manually give it STORAGE permissions but didn't work.

Although should be noted that I just updated Adaway to the most recent build since then (the previous version wasn't that old, just one version behind), so who knows, but regardless, I thank you for the information.
